sdl_android/bintray.gradle

```diff
@@ -0,0 +1,68 @@
+apply plugin: "com.jfrog.bintray"
+apply plugin: 'com.github.dcendents.android-maven'
+apply plugin: 'maven-publish'
+
+def siteUrl = 'https://github.com/smartdevicelink/sdl_android'      // Homepage URL of the library
+def gitUrl = 'https://github.com/smartdevicelink/sdl_android.git'   // Git repository URL
+group = "com.smartdevicelink" // Maven Group ID for the artifact
+def libDescription = 'SmartDeviceLink mobile library for Android'
+
+install {
+    repositories.mavenInstaller {
+        pom {
+            project {
+                packaging 'aar'
+
+                // Add your description here
+                name 'com.smartdevicelink:sdl_android'
+                description = libDescription
+                url siteUrl
+
+                // Set your license
+                licenses {
+                    license {
+                        name 'BSD 3-Clause'
+                        url 'https://opensource.org/licenses/BSD-3-Clause'
+                    }
+                }
+                scm {
+                    connection gitUrl
+                    developerConnection gitUrl
+                    url siteUrl
+                }
+            }
+        }
+    }
+}
+
+task sourcesJar(type: Jar) {
+    from android.sourceSets.main.java.srcDirs
+    classifier = 'sources'
+}
+
+artifacts {
+    archives sourcesJar
+}
+
+bintray {
+    // Authorization
+    user = ''
+    key = ''
+
+    configurations = ['archives']
+    pkg {
+        repo = 'sdl_android'
+        name = "sdl_android"
+        websiteUrl = siteUrl
+        vcsUrl = gitUrl
+        userOrg = 'smartdevicelink'
+        licenses = ["BSD 3-Clause"]
+        publish = true
+        version {
+            name = '4.3.0'  // Change to release version
```

Comment:
Is it possible to have this in a separate config? That way we can access it in the code.